common/djangoapps/student/models.py

```diff
@@ -1382,6 +1386,22 @@ def refundable(self):
         else:
             return True
 
+    def _refund_window_end_date(self):
```

Comment:
Also, you might as well make this a public method. I can see us wanting to display this value in the future. You'll also be able to appease PyLint without disabling checks.